docs/faq/index.rst

```diff
@@ -54,31 +54,41 @@ How does Flocker integrate with Kubernetes / Mesos / Deis / CoreOS / my favorite
    de
    facto
 
-Over time, we hope that Flocker becomes the de facto way for managing storage volumes with your favorite orchestration framework.  
-We are interested in expanding libswarm to include support for filesystems and are talking with the various open source projects about the best way to collaborate on storage and networking for volumes. 
+Over time, we hope that Flocker becomes the de facto way for managing storage volumes with your favorite orchestration framework.
+We are interested in expanding libswarm to include support for filesystems and are talking with the various open source projects about the best way to collaborate on storage and networking for volumes.
 If you'd like work with us on integration, get in touch on our Freenode IRC ``#clusterhq`` or `the Flocker Google group`_.
 You can also submit an issue or a pull request if you have a specific integration that you'd like to propose.
 
 If I clone a 2 GB database five times, won't I need a really large server with 10 GB of disk?
 *********************************************************************************************
 
-Thankfully no.  
-This is where ZFS makes things really cool. 
-Each clone is essentially free until the clone is modified. 
-This is because ZFS is a copy-on-write filesystem, so a clone is just a set of block pointers. 
+Thankfully no.
```

Comment:
These changes are quite annoying in a review - in `.py` files we don't have this issue because everything is checked by flake8 before it is in `master`. I'll have a look around for a reST linting tool.